tracing/AsyncHooks/problematic-modules.md

```diff
@@ -18,3 +18,4 @@ Module | Embedder API | Note
 [generic-pool](https://github.com/coopernurse/node-pool) | No | Used by a lot of database modules like for instance "pg"
 [mongoose](https://github.com/Automattic/mongoose/) | [#5929](https://github.com/Automattic/mongoose/issues/5929) | Most popular MongoDB ORM/ODM that implements its own user-land queueing.
 [graphql-subscriptions](https://github.com/apollographql/graphql-subscriptions/blob/26ed503566ecfab086e7530f0daf12b60ee3049c/src/pubsub-async-iterator.ts) | No | De-facto standard for working with GraphQL subscriptions. Custom queueing via an async iterator written in TypeScript and compiled down to ES5.
+[ioredis](https://github.com/luin/ioredis) | No | The only commonly used redis client with support for sentinels
```

Comment:
As a question is the other widely used redis package problematic (https://www.npmjs.com/package/redis) as well ?

Reply:
@mhdawson I have no idea actually. It doesn't support sentinels, so I can't really try it out at work.